packages/shopify-codemod/transforms/convert-default-export-objects-to-named-exports.js

```diff
@@ -0,0 +1,21 @@
+export default function convertDefaultExportObjectsToNamedExports({source}, {jscodeshift: j}, {printOptions = {quote: 'single'}}) {
+
+  return j(source)
+    .find(j.ExportDefaultDeclaration)
+    .forEach((nodePath) => {
+      const exportIndex = nodePath.parentPath.node.body.findIndex((prop) => prop.type === 'ExportDefaultDeclaration');
+      const body = nodePath.parentPath.value;
+      delete body[exportIndex];
+      const declarations = nodePath.node.declaration.properties;
```

Comment:
I prefer the `get` method for traversing: `nodePath.get('declaration', 'properties').node`